llvm/test/CodeGen/AMDGPU/propagate-waves-per-eu.ll

```diff
@@ -216,41 +216,30 @@ define internal i32 @bitcasted_function() {
   ret i32 0
 }
 
-define internal void @called_from_invalid_bounds_0() {
-; CHECK-LABEL: define internal void @called_from_invalid_bounds_0
-; CHECK-SAME: () #[[ATTR1]] {
-; CHECK-NEXT:    ret void
-;
+define internal void @called_without_min_waves() {
```

Comment:
I am not sure what the changes are trying to do in this test. Can you please explain?

I see that called_from_invalid_bounds() is called from called_from_invalid_bounds_1 but called_from_invalid_bounds_1 is not called from any kernel. Is that expected?

Reply:
I had renamed `kernel_invalid_bounds_0_8` with `called_from_invalid_bounds_1` since bounds [0,8] are no longer invalid. That's arguably a pretty non-sensical name for this and I can't remember why I picked this, so I will rename it.

> Is that expected?

Yes that is expected, though it's missing an attribute/accompanying CHECKs. Will address that as well.